README.md

```diff
@@ -28,3 +91,611 @@ Please find information on:
+  ```bash
+  docker run -it --rm itzg/rcon-cli --host 127.0.0.1 --port 25575 --password $RCON_PASSWORD
+  ```
+  and issue commands.
+
+- **SFTP**: If you included the `sshd-sftp.service` on port **2223**, you can do:
+  ```bash
+  sftp -P 2223 core@127.0.0.1
+  ```
+  and use your SSH key or local console-based authentication (if configured). You’ll see the `/home/core/papermc` directory for uploading mods/plugins.
+
+
+You can also use a GUI based program like [Filezilla](https://filezilla-project.org/). First create a new connection:
+
+![SFTP](./imgs/new_connection.png)
+
+Then fill out the necceserry fields:
+  1. In our case because we are running it locally, the `IP` is going to be localhost or simply `127.0.0.1`.
+  2. The port for `SFTP` (default is `2223` unless you have changed it)
+  3. Select the `Key file` option as Flatcar is passworldless
+  4. Select the `core` user
+  5. Select the key you downloaded or have locally that can be used to log into the `core` user on your VM
+
+Now you should be able to press `Connect` and connect to the server:
+
+![SFTP local connect](./imgs/sftp_connect_local.png)
+
+You should see a folder named `papermc`:
+
+![SFTP home](./imgs/sftp_home.png)
+
+This is where all of the game files exist and you can edit/download/ them upload new ones etc.:
+
+![SFTP papermc](./imgs/sftp_papermc.png)
+
+
+
+
+---
+
+### Additional Reading & Tips
+
+1. **Official Flatcar Docs**:
+   - [Running Flatcar with QEMU](https://www.flatcar.org/docs/latest/provisioning/booting-with-qemu/)
+   - [Ignition / Butane Usage](https://www.flatcar.org/docs/latest/provisioning/ignition/)
+
+2. **Port Forwarding**:
+   If you need more complex networking (e.g., bridging QEMU to your LAN instead of using user-mode forwarding), see the QEMU documentation or the Flatcar docs on advanced networking.
+
+3. **Performance**:
+   For best results, ensure your host has enough RAM and CPU to handle both your OS and the QEMU guest.
+
+4. **Dev & Testing**:
+   Running Flatcar + Minecraft in QEMU is perfect for **trying out** new Docker images or testing your config changes locally before deploying to a cloud environment.
+
+---
+
+**That’s it!** You’re now hosting your Minecraft server on a local Flatcar VM via QEMU. This approach makes it easy to experiment with different configurations, memory tiers, or plugin setups-all without incurring cloud costs. Have fun building your worlds! 🗺️
+
+
+## Example: Deploying on Azure Cloud ☁️
+
+Below is an **example** workflow for hosting this Flatcar Minecraft setup on **Microsoft Azure**. If you’re using another platform, you can skip the Azure commands and just launch a Flatcar instance normally with the same Ignition file. However, if you are new to Azure or **cloud computing**, the following explains each step in detail.
+
+---
+
+### 6.1 Connect to Azure & Understand the Cloud Shell
+
+1. **Open Azure Cloud Shell** in your browser, or use the **Azure CLI** locally.
```

Comment:
I suggest linking to the [Azure CLI](https://learn.microsoft.com/cli/azure/) docs here.

Reply:
Done

Reply:
Ah, but I deliberately gave you the link without `en-us` in it for a reason! That seems to automatically redirect to your language where possible. For me, it goes to `en-gb`.

Reply:
Done :)